core/src/Cuda/Kokkos_Cuda_Team.hpp

```diff
@@ -369,17 +369,19 @@ struct TeamVectorRangeBoundariesStruct<iType, CudaTeamMember> {
 template <typename iType>
 struct ThreadVectorRangeBoundariesStruct<iType, CudaTeamMember> {
   using index_type = iType;
+  const CudaTeamMember& member;
```

Comment:
Why a const reference instead of a copy?  (Not asking you to change, trying to figure out if that what discussed as part of the design)